public/locales/fr/translation.json

```diff
@@ -103,7 +103,8 @@
             "uninitialized": "Le scénario n'a pas encore été lancé"
           },
           "button": {
-            "downloadlogs": "Télécharger les logs"
+            "downloadlogs": "Télécharger les logs",
+            "refresh": "Rafraîchir"
```

Comment:
Since we can not add a tooltip when the button is disabled, display "Rafraîchir (sera ensuite indisponible pendant 15 secondes)"